sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactJurisdictionFlagsDto.java

```diff
@@ -20,19 +20,19 @@
 
 public class ContactJurisdictionFlagsDto implements Serializable {
 
-    private Boolean isInJurisdiction;
-    private Boolean isCaseInJurisdiction;
+    private final boolean isInJurisdiction;
```

Comment:
Why this refactoring? Is it always decidable without null as third option?

Reply:
removed from PR.